internal/extproc/watcher.go

```diff
@@ -65,45 +65,48 @@ func (cw *configWatcher) watch(ctx context.Context, tick time.Duration) {
 // loadConfig loads a new config from the given path and updates the Receiver by
 // calling the [Receiver.Load].
 func (cw *configWatcher) loadConfig(ctx context.Context) error {
+	var (
+		cfg *filterapi.Config
+		raw []byte
+	)
+
 	stat, err := os.Stat(cw.path)
+	if err != nil && os.IsNotExist(err) {
+		// If the file does not exist, do not fail (which could lead to the extproc process to terminate)
+		// Instead, load the default configuration and keep running unconfigured
+		cfg, raw = filterapi.MustLoadDefaultConfig()
```

Comment:
need to clear `err = nil`  now? not exist will end up captured in if err ! = nil below

Reply:
yes. when this method returned an error it made sense; not anymore. fixed